### ISSUE TITLE
Investigate tools for text-to-voice

### ISSUE THREAD

**Issue by mdconatser** [umgc/fall2022#232]
Consider entirely removing the custom screen reader in order to allow users with OS-level screen reader to be able to use that. Need to ensure pages work correctly with OS-level screen reader.

https://developer.apple.com/news/?id=v56qu1b3 (building in better support for native voiceover)

Consider alternate tools that may be better than the one currently implemented in the application.

Document findings of approaches considered, and why they should be selected or excluded as we need this information in the TDD. Place findings in the documents as a draft, no need for extensive polishing within this task. Some content may be moved out later to the Programmers Guide.

Additionally, if implementation appears to be simple, developing a proof of concept (or locating an existing one built by someone else) would also be great for our next presentation.

**Comment by mdconatser** [comment]
from the professor in response to me asking about if it would be acceptable to remove the "custom" screen reader and recommend our applications just ensure that we support the native OS accessibility tools: "The UI/UX is left to the team. The higher-level requirement is that the app should be easy to use be someone who is visually impaired as well as someone who is not. If the team believes that using the OS Screen Reader is more appropriate, please note the reasoning behind your decision, proceed with it, and then show the client the improved interface."

**Comment by mdconatser** [comment]
thinking we should continue pursuing removing the current implementation and enhancing support for the native iOS "VoiceOver" and android equivalent.

**Comment by Minyahil40** [comment]
added a document under Milestone 2 folder for review. 

[https://umgcdev361.sharepoint.com/:w:/r/sites/SWEN670Fall2022/Shared%20Documents/Team%20B%20Channel/Milestone%202%20(SAT%20SEP%2017)/Flutter%20Accessibility%20Support.docx?d=w8fa1a3f2e536449e99f70e51ed6900e1&csf=1&web=1&e=TxgvPk](url)
